db/sql/prototype.sql

```diff
@@ -0,0 +1,491 @@
+/*
+Functions for testing feasibility of moving different Mathesar pieces to the database. These are
+part of the 'remove SQLAlchemy' project.
+
+There are three involved schemas:
+      __msar:  These functions aren't designed to be used except by other Mathesar functions.
+               Generally need preformatted strings as input, won't do quoting, etc.
+        msar:  These functions are designed to be used more easily. They'll format strings, quote
+               identifiers, and so on.
+  msar_views:  This schema is where internal mathesar views will be stored.
+
+The reason they're so abbreviated is to avoid namespace clashes, and also because making them longer
+would make using them quite tedious, since they're everywhere.
+*/
+
+CREATE SCHEMA IF NOT EXISTS __msar;
+CREATE SCHEMA IF NOT EXISTS msar;
+CREATE SCHEMA IF NOT EXISTS msar_views;
+
+----------------------------------------------------------------------------------------------------
+----------------------------------------------------------------------------------------------------
+-- GENERAL DDL FUNCTIONS
+--
+-- Functions in this section are quite general, and are the basis of the others.
+----------------------------------------------------------------------------------------------------
+----------------------------------------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION
+__msar.exec_ddl(command text) RETURNS text AS $$/*
+Execute the given command, returning the command executed.
+
+Not useful for SELECTing from tables. Most useful when you're performing DDL.
+
+Args:
+  command: Raw string that will be executed as a command.
+*/
+BEGIN
+  EXECUTE command;
+  RETURN command;
```

Comment:
What's the motivation for returning the string representation of a command that was executed? Not criticizing, just wondering.

Reply:
I think this will make testing and debugging easier. No other motivation. We don't really need this string for anything. It might be safer to remove it (I.e., avoid inadvertently picking up something sensitive in a log somewhere)